public/app/assetlibrary/item/assetLibraryItemController.js

```diff
@@ -33,14 +33,54 @@
      */
     var getCurrentAsset = function() {
       assetLibraryItemFactory.getAsset(assetId).success(function(asset) {
-        // Ensure that the newest comments are at the top
-        // TODO: Add sorting to allow for replies
-        asset.comments = asset.comments.reverse();
+
+        // Build the comment tree. First, the top level comments
```

Comment:
I haven't looked at the backend logic yet, but should anything be done here to ensure that the child comments are sorted chronologically?